tests/clientEncryption/clientEncryption-rewrapManyDataKey-003.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+MongoDB\Driver\ClientEncryption::rewrapManyDataKey() accepts null for $options
```

Comment:
FYI: this test covers the changes from https://github.com/mongodb/mongo-php-driver/pull/1347 for [PHPC-2124](https://jira.mongodb.org/browse/PHPC-2124). I don't believe we had existing tests for passing `null`, as we probably would have caught the stub conflict sooner if so.